### PR TITLE
#151476592 fix landing page links

### PR DESCRIPTION
**Pull request by gbengaoyetade**
#### What does this PR do?
Makes each link on the landing page conform to its reference

#### Description of Task to be completed?
Users should be able to link from the landing page to other links provided on the landing page.

#### How should this be manually tested?
Visit the landing page of the app and click each link

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
Connect each link on the landing page to it's destination

### CODE REVIEW

File: app/views/includes/head.jade [hunk 1]
```diff
@@ -13,9 +13,11 @@ head
 
   link(href='/img/icons/favicon.ico', rel='shortcut icon', type='image/x-icon')
   link(href='https://fonts.googleapis.com/css?family=Lobster', rel='stylesheet', type='text/css')
-  link(rel="stylesheet" href='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.2/css/materialize.min.css')
+  link(rel="stylesheet" href='https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/css/materialize.min.css')
   link(rel="stylesheet" href='https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css')
   link(href='https://fonts.googleapis.com/icon?family=Material+Icons', rel='stylesheet', type='text/css')
+  link(rel="stylesheet" href="../css/cfh.css")
+  //- link(rel="stylesheet" href="../css/style.css")
```
Comment (enodi) [inline app/views/includes/head.jade]: remove unused code

File: public/css/style.css [hunk 2]
```diff
@@ -5,17 +5,17 @@
 html {
   font-family: "Montserrat", sans-serif;
 }
-header .nav-wrapper a {
+/* header .nav-wrapper a {
 font-family: 'Satisfy'
-}
+} */
```
Comment (enodi) [inline public/css/style.css]: remove unused code

File: public/views/index.html [hunk 3]
```diff
@@ -233,10 +225,9 @@ <h3 class="center"><a target="_blank"  href="http://github.com/">Segun</a></h3>
         </div>
       </footer>
       <!-- Latest compiled and minified Bootstrap JavaScript and Jquery-->
-      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+      <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script> -->
```
Comment (enodi) [inline public/views/index.html]: remove unused code

**Review by enodi — CHANGES_REQUESTED** [review]

File: app/views/includes/head.jade (hunk 1 above)
Comment (segunolalive) [inline app/views/includes/head.jade]: remove this line.

style.css is included in another line

**Review by segunolalive — CHANGES_REQUESTED** [review]

**Review by enodi — APPROVED** [review]

**Review by segunolalive — APPROVED** [review]

**Review by ibonly — APPROVED** [review]

**Pull request merged**